CONTRIBUTING.md

```diff
@@ -347,7 +347,7 @@ curl -sL https://github.com/SigNoz/signoz/raw/develop/sample-apps/hotrod/hotrod-
 ```bash
 kubectl -n sample-application run strzal --image=djbingham/curl \
   --restart='OnFailure' -i --tty --rm --command -- curl -X POST -F \
-  'locust_count=6' -F 'hatch_rate=2' http://locust-master:8089/swarm
+  'user_count=2' -F 'spawn_rate=1' http://locust-master:8089/swarm
```

Comment:
Use the same as the one in other places such that contributor would get same load as that of user installing the sample app.

```suggestion
  'user_count=6' -F 'spawn_rate=2' http://locust-master:8089/swarm
```